src/App.js

```diff
@@ -1,26 +1,49 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
+import { getMovie } from './api/movies';
 
-export class App extends Component {
-  state = {
-    movies: data,
+export const App = () => {
+  const [movies, setMovies] = useState(data);
+  const [query, setQuery] = useState('');
+
+  const changeQuery = (value) => {
+    setQuery(value);
   };
```

Comment:
Do you need this wrapper function? It's add no functionality to the setQuery, it's work exactly same way